openhands/agent_server/desktop_service.py

```diff
@@ -203,7 +203,11 @@ def get_desktop_service() -> DesktopService | None:
     global _desktop_service
     config = get_default_config()
 
-    if not config.enable_vnc:
+    if not config.enable_vnc and os.getenv("ENABLE_VNC", "").lower() not in (
+        "1",
+        "true",
+        "yes",
+    ):
```

Comment:
Shouldn't we be able to load this ENABLE_VNC to the config? 🤔 Why would this happen?